performance/performance.md

```diff
@@ -40,6 +40,15 @@ A good example is a social media widget. A plugin may call out to a third-party
 
 Persistent object caching can speed up WordPress and is offered my many managed WordPress hosts or can be set up independently. You will also need a plugin to connect WordPress to the object cache. There are many [available on the plugin directory](https://wordpress.org/plugins/search/object+cache/)
 
+### Op-code Caching (a.k.a server-side caching)
+As mentioned in the section on full page caching, WordPress processes PHP scripts as part of producing the HTML, CSS and JavaScript for a browser to load. It takes time for the web server to read each PHP script WordPress needs, to compile the script and to run the PHP script. By default, the web server has to do this process for every single visit to every single page. Full page caching can reduce how much the web server has to do this depending on what kind of full page caching is used, but there may still be some PHP scripts that have to be read, compiled, and run for every single visit.
```

Comment:
> there may still be some PHP scripts that have to be read, compiled, and run for every single visit

Can you list a few examples of these? I think that will help people understand when it's important to consider op-code caching